### PR TITLE
Apply android-sdk-versions-plugin

### PR DESCRIPTION
**Pull request by harvsu**
This is to apply the plugin that validates and persists SDK version in a file in assets folder.
More context to this is in: 
https://github.com/mapbox/mobile-telemetry/issues/452
https://github.com/mapbox/mobile-telemetry/issues/423
https://github.com/mapbox/android-sdk-versions-plugin

### CODE REVIEW

**Comment by harvsu** [comment]
@tobrun @mapbox/maps-android  Since this is closed by the bot, I want to bring to your attention that this change in Maps is necessary for reformed user agent in Telemetry. Please let us know if this is handled in some other commit/pull request.

**Comment by alfwatt** [comment]
@tobrun any update since "tobrun moved this from Next to In progress in Android Maps on Mar 4"?

File: gradle/dependencies.gradle [hunk 1]
```diff
@@ -7,35 +7,36 @@ ext {
...
+            artifactory      : '4.9.3',
+            androidPublish   : '3.6.2',
+            lint             : '26.4.2',
+            gms              : '16.0.0',
+            soLoader         : '0.8.0',
+            jacoco           : '0.8.4',
+            appcenter        : '1.4',
+            ktlint           : '0.34.0',
+            commonsIO        : '2.6',
+            mapboxSdkVersions: '1.0.1'
```
Comment (tobrun) [inline gradle/dependencies.gradle:39]: fixup unintended formatting

**Comment by LukasPaczos** [comment]
Is there a chance that this PR added a transitive Jacoco dependency? That's what I'm noticing when bumping Maps dependency downstream https://github.com/mapbox/mapbox-navigation-android/pull/3267.

**Comment by LukasPaczos** [comment]
Hmm, I think that was an issue with my cache.

**Pull request merged**